vendor/assets/javascripts/jquery.pjax.js

```diff
@@ -293,7 +293,7 @@ function pjax(options) {
     }
 
     // Only blur the focus if the focused element is within the container.
-    var blurFocus = $.contains(context, document.activeElement)
+    var blurFocus = $.contains(context.get(0), document.activeElement)
```

Comment:
We don't make changes in this file, we take it from [upstream repo](https://github.com/defunkt/jquery-pjax). If you found a bug in pjax send it to them.